app/services/supplementary_data.py

```diff
@@ -64,6 +66,13 @@ def get_supplementary_data_v1(
         backoff_factor=SUPPLEMENTARY_DATA_REQUEST_BACKOFF_FACTOR,
     )
 
+    # Type ignore: oidc_credentials_service is a singleton of this application
+    oidc_credentials_service: OIDCCredentialsService = current_app.eq["oidc_credentials_service"]  # type: ignore
+    credentials = oidc_credentials_service.get_credentials(
+        iap_client_id=SDS_OAUTH2_CLIENT_ID
+    )
```

Comment:
It would be clearer to raise an exception if we don't have a client id than the request failing due to passing None.